src/map/itemdb.hpp

```diff
@@ -3139,6 +3145,7 @@ struct s_item_group_entry
 struct s_item_group_random
 {
 	uint32 total_rate;
+	uint16 algorithm;
```

Comment:
```suggestion
	e_item_subgroup_algorithm algorithm;
```

Reply:
In my opinion, we shouldn't add a second enum for the algorithm at all but instead use the existing one `e_group_search_type`.

https://github.com/rathena/rathena/pull/8849/files

```
/// Enum for different ways to search an item group
enum e_group_search_type : uint8 {
	GROUP_SEARCH_BOX = 0, // Always return an item from the group, rate determines which item is more likely to be returned
	GROUP_SEARCH_PICK = 1, // Rate is the amount of items in the group, return a random item and remove it from the group
	GROUP_SEARCH_DROP = 2, // Pick one item from the group and check use rate as drop rate, on fail, do not return any item
};
```